apps/studio/components/interfaces/TableGridEditor/GridHeaderActions.tsx

```diff
@@ -132,28 +169,70 @@ const GridHeaderActions = ({ table }: GridHeaderActionsProps) => {
             </Tooltip.Portal>
           </Tooltip.Root>
         )}
-
-        {(table.rls_enabled || showRLSWarning) && (
-          <Button
-            asChild
-            type={table.rls_enabled ? 'link' : 'warning'}
-            icon={
-              table.rls_enabled ? (
-                <IconLock strokeWidth={2} size={14} />
+        <div>
+          {(table.rls_enabled || showRLSWarning) && (
+            <>
+              {table.rls_enabled ? (
+                <div className="flex items-center gap-1">
+                  <Link passHref href={`/project/${projectRef}/auth/policies?search=${table.id}`}>
+                    <Button
+                      type={table.rls_enabled ? 'default' : 'warning'}
+                      className="group"
+                      icon={
+                        policies.length > 0 ? (
+                          <span>
```

Comment:
Span in a span

Reply:
annoyingly addressed with !h-[28px] !py-0